### DISCUSSION TITLE
ETR for a new release?

### DISCUSSION THREAD

**Discussion by iantoal** [aws/credentials-fetcher#83] [General]
Hi guys, I  noticed that it's been quite some time since you did a release of the package and I was wondering if a release was imminent or what is your criteria?
I have a pressing need for the [change from June where you can specify a domain controller](https://github.com/aws/credentials-fetcher/commit/2c5eda69107e4901df2e557103487abd1f725681) as it will fix some challenges I have with routing and availability of on-prem domain controllers during a migration that is currently happening.  I know I can roll my own package, but I'd prefer not to have to do that right now if you guys are planning a release imminently/soon. Thanks, Ian

**Comment by saikiranakula-amzn** [comment]
Hi Ian, Thank you for reaching out to us. We are actively working on the release and expect it in early December if everything goes as planned.

**Comment by smhmhmd** [comment]
@iantoal 
We have a new release, please let us know if this resolves your issues.
https://github.com/aws/credentials-fetcher/releases/tag/v1.3.2

CC: @saikiranakula-amzn @sb-ruisms
